src/server/active_tran_server.hpp

```diff
@@ -56,7 +53,7 @@ class active_tran_server
     void push_request (ats_to_ps_request reqid, std::string &&payload);
 
   private:
-    using page_server_request_autosend = cubcomm::request_queue_autosend<page_server_request_queue>;
+    using ps_t = cubcomm::request_sync_client_server<ats_to_ps_request, ps_to_ats_request, std::string>;
```

Comment:
Can "ps" be replaced by a more verbose name? page_server_conn, page_server_request_cs, page_server_link, or any name you find fit :).

Reply:
Done